plugins/org.jkiss.dbeaver.net.ssh/src/org/jkiss/dbeaver/model/net/ssh/SSHUtils.java

```diff
@@ -293,6 +294,11 @@ private static SSHHostConfiguration loadHostConfiguration(
         return new SSHHostConfiguration(username, hostname, port, auth);
     }
 
+    @NotNull
+    public static String getKeyData(String keyValue) {
```

Comment:
It doesn't really "get" data from the input but rather sanitizes/normalizes the input.

Reply:
javadoc was added